### ISSUE TITLE
JAWS does not work properly

### ISSUE THREAD

**Issue by ognil** [wet-boew/wet-boew#8088]
Hi guys,

New version of the script wet-boew.min does not include 
aria-posinset  and aria-setsize as follow (it is from old version  v4.0.22 - 2016-08-11) :

`  <li><a role="menuitem " tabindex="-1"  aria-posinset="2" aria-setsize="15"
 href="/24-hour-emergency-numbers">24 hour Emergency `Numbers</a></li>`

As a result, JAWS repeats at least two times the name of the menu item and does not says “two of fifteen”. 

George

**Comment by nschonni** [comment]
Duplicate of #8081
